File: Year-2019/Mar/2019-03-12-242/watercow/242.c
class Solution {
public:
	bool isAnagram(string s, string t) {
		vector<int> count(26, 0);

		for (auto i : s) ++count[i - 'a'];
		for (auto i : t) --count[i - 'a'];

		for (auto i : count) {
			if (i != 0) return false;
		}

		return true;
	}
};

File: Year-2019/Mar/2019-03-12-242/lb/242.java
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] sc =  s.toCharArray();
        char[] tc = t.toCharArray();
        if (sc.length!=tc.length){
            return false;
        }
        int[] si = new int[26];
        int[] ti = new int[26];
        for(int i=0;i<sc.length;i++){
            si[sc[i] - 'a'] ++;
            ti[tc[i] - 'a'] ++;
        }
        for (int i = 0;i<26;i++){
            if (si[i] != ti[i])
                return false;
        }
        return true;
    }
}
